build.gradle

```diff
@@ -398,7 +398,6 @@ task createApiDocsRedirect {
 }
 
 task publishDocs {
-  dependsOn 'closeAndReleaseRepository'
```

Comment:
This task does not appear to be defined. I left the manual instructions that seem to accomplish its intended function.